packages/dids/package.json

```diff
@@ -89,7 +89,8 @@
   "devDependencies": {
     "@playwright/test": "1.40.1",
     "@types/bencode": "2.0.4",
-    "@types/chai": "4.3.6",
+    "@types/chai": "4.3.16",
+    "@types/chai-as-promised": "7.1.8",
```

Comment:
This was intentionally removed in a prior PR so that we can eventually migrate off Mocha & Chai.
```suggestion
```